services/datamanager/datasync/sync.go

```diff
@@ -158,7 +159,12 @@ func (s *syncer) syncDataCaptureFile(f *datacapture.File) {
 		func(ctx context.Context) error {
 			err := uploadDataCaptureFile(ctx, s.client, f, s.partID)
 			if err != nil {
-				s.logger.Errorw(fmt.Sprintf("error uploading file %s", f.GetPath()), "error", err)
+				if errors.Is(err, context.Canceled) {
+					s.logger.Debugw(fmt.Sprintf("context cancelled while uploading file %s", f.GetPath()),
```

Comment:
[nit] Since the error text will say 'context canceled' this will be somewhat redundant. Can either remove your "context cancelled" text or the error message.

Reply:
Good call! Changed to match generic error message below, so now it's the same but just with the lower log level